resqpy/surface/_surface.py

```diff
@@ -636,7 +636,7 @@ def set_from_point_set(self,
             assert -90.0 < saucer_parameter < 90.0, f'simple saucer angle parameter must be less than 90 degrees; too big: {saucer_parameter}'
             simple_saucer_angle = saucer_parameter
             saucer_parameter = None
-        assert saucer_parameter is None or 0.0 <= saucer_parameter < 1.0
+        assert saucer_parameter is None, 'saucer_parameter no longer supported, use simple_saucer_angle'
```

Comment:
My mistake here, the argument is still saucer_parameter. We can remove this assertion completely.